pkg/client/unversioned/portforward/portforward_test.go

```diff
@@ -206,7 +206,7 @@ func TestGetListener(t *testing.T) {
 }
 
 // fakePortForwarder simulates port forwarding for testing. It implements
-// kubeletserver.PortForwarder.
+// portforward.PortForwarder.
```

Comment:
That already exists a few lines down. IMO it's fine to have both docs and the actual static compile check.